.github/workflows/update-generated.yaml

```diff
@@ -35,6 +35,14 @@ jobs:
         role-duration-seconds: 900
         role-session-name: eksctl-update-coredns-assets
         role-to-assume: ${{ secrets.UPDATE_COREDNS_ROLE_ARN }}
+    - name: Configure AWS credentials for ec2 info update
+      if: ${{ matrix.resource == 'ec2-info' }}
+      uses: aws-actions/configure-aws-credentials@ececac1a45f3b08a01d2dd070d28d111c5fe6722 # v4.1.0
+      with:
+        aws-region: us-west-2
+        role-duration-seconds: 900
+        role-session-name: eksctl-update-ec2-info-assets
+        role-to-assume: ${{ secrets.UPDATE_EC2_INFO_ROLE_ARN }}
```

Comment:
could probably just use one role that has list/describe permissions - if we need to scope it, we have session tags that can be used

Reply:
we also need to ensure this doesn't run in an account that has preview access to new instance types that are not yet released